### PR TITLE
Initial Data Loading code

### PR DESCRIPTION
**Pull request by dylandaniels**
Converts the data into a pandas.DataFrame object for easier processing.

### CODE REVIEW

**Comment by dylandaniels** [comment]
@peterasujan please review!

File: mousestyles/data/__init__.py [hunk 1]
```diff
@@ -1,15 +1,46 @@
...
+def load_all_features():
+    """
+    Returns a (21131, 13) size pandas.DataFrame object corresponding to
+    9 features over each mouse's 2-hour time bin. The first four columns
+    index each mouses's 2-hour bin:
+
+    Column 0: the strain of the mouse (0-15)
+    Column 1: the mouse number (number depends on strain)
+    Column 2: the day number (5-16)
+    Column 3:
```
Comment (peterasujan) [inline mousestyles/data/__init__.py]: Column 3: hour of day?

File: mousestyles/data/__init__.py [hunk 2]
```diff
@@ -1,15 +1,46 @@
...
+    Column 0: the strain of the mouse (0-15)
+    Column 1: the mouse number (number depends on strain)
+    Column 2: the day number (5-16)
+    Column 3: 
 
-    feat_arr = ['ASProbability', 'ASNumbers', 'ASDurations', 'Food', 'Water', 'Distance', 'ASFoodIntensity', 'ASWaterIntensity', 'MoveASIntensity']
-    feat_arr_units = ['Active state probability', 'Number AS onsets', 'Total time [sec]', 'Food consumed [g]', 'Water consumed [mg]', 'Distance travelled [m]', 'ASFoodIntensity', 'ASWaterIntensity', 'MoveASIntensity [cm/ASsec]']
+    The remaining 9 columns are the computed features.
+
+    Returns
```
Comment (peterasujan) [inline mousestyles/data/__init__.py]: We should settle on a docstring convention tomorrow and make sure that it's clearly outlined in the contributor doc, as you suggested in #17.

**Comment by peterasujan** [comment]
Good stuff, @dylandaniels! My main comment would be to resolve the style/linting errors for the two .py files. I've also left a couple line comments on the diff.

File: mousestyles/data/__init__.py [hunk 3]
```diff
@@ -1,15 +1,46 @@
...
+    features_data_frame : pandas.DataFrame
+        A dataframe of computed features.
+    """
+    features = ['ASProbability', 'ASNumbers', 'ASDurations', 'Food', 'Water', 'Distance', 'ASFoodIntensity', 'ASWaterIntensity', 'MoveASIntensity']
+
+    all_features = np.load(_os.path.join(data_dir, 'all_features_mousedays_11bins.npy'))  # 9 x 1921 x (3 labels + 11 feature time bins)
+   
+    # Here we begin reshaping the 3-d numpy array into a pandas 2-d dataframe
+    columns = ['strain', 'mouse', 'day']
+    for i in range(11):
```
Comment (peterasujan) [inline mousestyles/data/__init__.py]: This whole loop could be replaced with `columns += list(range(11))` (as you know, I love avoiding loops at all costs :stuck_out_tongue_winking_eye:)

**Comment by dylandaniels** [comment]
There are still a couple styles issues to fix--I'll incorporate them later but we should merge this in for now.

**Comment by peterasujan** [comment]
To be most proper, this should probably be rebased to remove the need for the merge commit (4d5a8c4).

**Comment by peterasujan** [comment]
Looks good, thanks!

**Pull request merged**